modules/persistent-storage-csi-efs-troubleshooting.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-csi-aws-efs.adoc
+
+[id="efs-troubleshooting_{context}"]
+= AWS EFS troubleshooting
+
+The following information provides guidance on how to troubleshoot issues with AWS EFS:
+
+* The AWS EFS Operator and CSI driver run in namespace `openshift-cluster-csi-drivers`.
+
+* To initiate gathering of logs of the AWS EFS Operator and CSI driver, run the following command:
++
+[source, screen]
```

Comment:
```suggestion
[source,terminal]
```

We don't use `[source,screen]`. 

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output